app/javascript/app/platform-authenticator.js

```diff
@@ -1,12 +1,18 @@
+function postPlatformAuthenticator(userIntent) {
+  const xhr = new XMLHttpRequest();
+  xhr.open('POST', '/analytics', true);
+  xhr.setRequestHeader('Content-type', 'application/x-www-form-urlencoded');
+  xhr.send(`platform_authenticator[available]=${userIntent}`);
+}
 function platformAuthenticator() {
-  const isEnabled = document.querySelector('[data-platform-authenticator-enabled]');
-  if (isEnabled && window.PublicKeyCredential) {
+  if (document.querySelector('[data-platform-authenticator-enabled]')) {
+    if (!window.PublicKeyCredential) {
+      postPlatformAuthenticator(false);
+      return;
+    }
```

Comment:
Style: rather than an explicit `return`, I'd wrap the implicit "else" in an explicit `else { ... }` to make it more obvious that it won't be reached if the condition is false.